base/datafmt.jl

```diff
@@ -277,8 +273,6 @@ function readdlm_string(sbuff::ByteString, dlm::Char, T::Type, eol::Char, auto::
         catch ex
             if isa(ex, TypeError) && (ex.func == :store_cell)
                 T = ex.expected
-            elseif get(optsd, :ignore_invalid_chars, false)
```

Comment:
Actually, this whole option doesn't really make sense, which is why I deleted it. The transformation is now a no-op.